src/dotnet/APIView/APIView/Model/CodeFile.cs

```diff
@@ -41,7 +42,9 @@ public string VersionString
         public string ServiceName { get; set; }
         public string PackageDisplayName { get; set; }
         public string PackageVersion { get; set; }
+        // Deprecated: Use CrossLanguageMetadata.CrossLanguagePackageId instead. Kept for backward compatibility with TypeSpec parsers.
```

Comment:
I think the only parser that has this implemented is Python. If that is true, once Python modifies their parser, then you should be able to remove this